biopandas/pdb/pandas_pdb.py

```diff
@@ -127,6 +128,23 @@ def fetch_pdb(self, pdb_code):
         self._df = self._construct_df(pdb_lines=self.pdb_text.splitlines(True))
         return self
 
+    def fetch_af2(self, uniprot_id: str):
```

Comment:
Same here, should we just add a `source="..."` parameter to `fetch_pdb` instead?